src/xdocs/checks/javadoc/javadocmissingleadingasterisk.xml

```diff
@@ -35,14 +35,11 @@
             </tr>
             <tr>
               <td>violateExecutionOnNonTightHtml</td>
-              <td>
-                Control when to print violations if the Javadoc being examined by this check
-                violates the tight html rules defined at
-                <a href="../../writingjavadocchecks.html#Tight-HTML_rules">Tight-HTML Rules</a>.
-              </td>
+              <td>Control when to print violations if the Javadoc being examined by this check violates the tight html rules defined at <a href="../../writingjavadocchecks.html#Tight-HTML_rules">
+     Tight-HTML Rules</a>.</td>
               <td><a href="../../property_types.html#boolean">boolean</a></td>
               <td><code>false</code></td>
-              <td>8.38</td>
+              <td>8.3</td>
```

Comment:
Same.

Reply:
workaround-ed.
actual fix will be https://github.com/checkstyle/checkstyle/issues/14052

Reply:
same.

Reply:
now, link is added at https://github.com/checkstyle/checkstyle/pull/14045/files#diff-582038065de9c242c07931401cfa262fcfd745638bf7c42736dba5793a3294cdR173
to avoid conflict, let not add it here also.